core/domain/exp_services.py

```diff
@@ -656,6 +656,10 @@ def apply_change_list(exploration_id, change_list):
                 if (change.property_name ==
                         exp_domain.STATE_PROPERTY_PARAM_CHANGES):
                     state.update_param_changes(change.new_value)
+                elif (
+                        change.property_name ==
```

Comment:
Follow the formatting of the previous branch (move this line to the previous one, etc.)